frontend/packages/dev-console/integration-tests/features/Helm/actions-on-HelmRelease.feature

```diff
@@ -0,0 +1,35 @@
+Feature: Helm Chart
+    User should able to perform the actions on the helm releases
+
+Background:
+    Given user logged into the openshift application
+    And user is at developer perspecitve
+
+
+@regression, @smoke
+Scenario: Perform Upgrade action on Helm Release through Context Menu: HR-08-TC02
+    Given user right clicks on the Helm Release
+    When user clicks on the Upgrade action
+    And user updates the chart Version
+    And user clicks on the upgrade button
+    Then the helm release should get upgradaed 
+    And user should get redirected to topology page
```

Comment:
Don't use assertion words like should, verify etc.. in  Then statements

Reply:
Updated accordingly.